spec/run_oc_pedant.rb

```diff
@@ -88,10 +107,12 @@ def start_server(chef_repo_path)
     '--skip-cookbook-artifacts',
     '--skip-containers',
     '--skip-api-v1'
+  ] + chef_fs_skips)
 
-  ])
+  fail_fast = []
+  # fail_fast = ["--fail-fast"]
```

Comment:
I think it makes sense for developers to manually turn this on when needed. In Ci you probably want to see all the failures, for example.